File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer,Integer> mp = new HashMap<>();
        for(int num :nums){
            mp.put(num,mp.getOrDefault(num,0)+1);
            if(mp.get(num)>1) return true;
        }
        return false;

        
    }
}

File: 217-contains-duplicate/contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
         freq={}
         for x in nums:
            freq[x]= freq.get(x,0)+1
         for x in nums:
            if(freq[x]>1) :
                return True
         return False
         
        